src/tests/ftest/cart/util/cart_utils.py

```diff
@@ -75,6 +77,11 @@ def wait_process(proc, wait_time):
 
         return procrtn
 
+    @staticmethod
+    def cleanup_processes():
+        """ Clean up cart processes, in case avocado/apricot does not. """
+        stop_processes(["localhost"], "'(crt_launch|orterun)'")
```

Comment:
As mentioned above, it would be good to run another `pgrep` to verify if this was successful and return an error message (to be handled by https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/apricot/apricot/test.py#L270) if processes were still running.

Reply:
Let's address this in a separate PR.